client/src/Components/Navbar/index.css

```diff
@@ -0,0 +1,66 @@
+.FloatingButton {
+  position: fixed;
+  bottom: 10px;
+  right: 10px;
+  background-color: #671e75;
+  outline: none;
+  border: 2px solid #00acd0;
```

Comment:
I don't think this border fits with their branding, i think just the rounded purple background is fine